cmd/conformance-tests/scenarios_kubevirt.go

```diff
@@ -82,7 +82,7 @@ func (s *kubevirtScenario) Cluster(secrets secrets) *apimodels.CreateClusterSpec
 
 func (s *kubevirtScenario) NodeDeployments(_ context.Context, num int, _ secrets) ([]apimodels.NodeDeployment, error) {
 	var sourceURL string
-	registryAddr := "http://http-server.kube-system.svc.cluster.local"
+	registryAddr := "http://10.244.2.19"
```

Comment:
should be `10.244.1.19`

Reply:
Thanks! did not check `master`, only the PR.